### ISSUE TITLE
[DEPR-2U] Node 18 - edx-platform

### ISSUE THREAD

**Issue by timmc-edx** [edx/edx-arch-experiments#727]
2U followup ticket.

- Original: https://github.com/openedx/edx-platform/issues/35015
- Didn't see a precise date on the original, but it sounds like Axim is targeting December for removal.

This will need a PR in [edx/configuration](https://github.com/edx/configuration) repo to upgrade the Node version. For reference, this was created last time to upgrade the Node version to 18, https://github.com/openedx-unsupported/configuration/pull/7153

```[tasklist]
### Tasks
- [ ] Create a sandbox with the configuration branch to test Node 20
- [ ] After testing on the sandbox, merge the PR to deploy it on stage, production, and edge.
```

**Comment by timmc-edx** [comment]
@jristau1984 Should this go to Arbi-BOM?

**Comment by jristau1984** [comment]
I have moved this to Arbi-BOM.

**Comment by UsamaSadiq** [comment]
The Configuration upgrade PR is currently blocked by Axim and is scheduled to be merged after December.

**Comment by feanil** [comment]
FYI, we've dropped the Node 18 testing in edx-platform now that it's December, per the deadline in the DEPR.
